TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/automation/TelepresenceBotAccessibilityService.java

```diff
@@ -0,0 +1,54 @@
+package com.novoda.tpbot.automation;
+
+import android.accessibilityservice.AccessibilityService;
+import android.util.Log;
+import android.view.accessibility.AccessibilityEvent;
+import android.view.accessibility.AccessibilityNodeInfo;
+
+import java.util.List;
+
+public class TelepresenceBotAccessibilityService extends AccessibilityService {
```

Comment:
I personally find it quite disturbing the number of things you can do with this `Service`. It is scary! 😨